### ISSUE TITLE
Operator `+` on ranges throws runtime exception `AbsoluteCellRange: Wrong range size`

### ISSUE THREAD

**Issue by BrianHung** [handsontable/hyperformula#1267]
### Description

Encountered an internal error while trying some array formula tests. The specific formula `"='Sheet1'!A:A+'Sheet1'!1:1"` I expect to result in a `DetailedCellError` instead.

```ts
import HyperFormula from "hyperformula";

console.log(
  `%c Using HyperFormula ${HyperFormula.version}`,
  "color: blue; font-weight: bold"
);

const hf = HyperFormula.buildFromSheets({
  "Sheet1": [["1"]],
}, {
  licenseKey: "gpl-v3",
  useArrayArithmetic: true,
});

hf.setCellContents({ sheet: 0, row: 0, col: 0 }, "='Sheet1'!A:A+'Sheet1'!1:1")

const sheetName = "Sheet1";
const sheetId = hf.getSheetId(sheetName);
```

### Demo
https://codesandbox.io/s/red-field-hmeldt?file=/src/hyperformulaConfig.js

**Comment by sequba** [comment]
@BrianHung thank you for reporting this issue. You are right, it should be handled by returning a cell error. We'll work on it in one of the upcoming releases.

**Comment by AMBudnik** [comment]
Hi @BrianHung 

I have good news! We're closing this issue as solved and released in HyperFormula 2.6.1.
Here's a list of all the changes https://github.com/handsontable/hyperformula/releases/tag/2.6.1
